core/overlays/test/pgweb.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: pgweb
+spec:
+  lookupPolicy:
+    local: false
+  tags:
+  - annotations: null
+    from:
+      kind: DockerImage
+      name: sosedoff/pgweb:0.11.3
+    name: latest
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  labels:
+    service: pgweb
+    app: thoth
+    component: pgweb
+  name: pgweb
+spec:
+  replicas: 1
+  selector:
```

Comment:
https://github.com/thoth-station/thoth-application/blob/95d6dea7e6015c5e37dd9740c0f72695f64d5a4f/core/overlays/test/kustomization.yaml#L3 will add more labels, so the selector and the template labels need to include them. feel free to remove the version commonLabel ;)

Reply:
Added the commonlabel to service's selector and template's label.